### PR TITLE
ci: prevent test fail on unrelated dep change

### PR DESCRIPTION
**Pull request by vthemelis**
Fixes https://github.com/widgetti/ipyaggrid/issues/44

@maartenbreddels, does this comply with the first phase of your view?

If you're not opposed to it, I can also add a tox file with command lines for running the tests and performing lock file upgrading.

### CODE REVIEW

**Comment by mariobuikhuizen** [comment]
Thanks for this fix @vthemelis! pip-compile was new to me and seems to work really nice!

> @maartenbreddels, does this comply with the first phase of your view?

yes, this does comply!

> If you're not opposed to it, I can also add a tox file with command lines for running the tests and performing lock file upgrading.

We're still ironing out the details of automating the updating of the lock file in CI and using that in all our projects. We want to implement this in the near future for ipyaggrid first. I think we should merge this PR first without adding extra functionality, so the other PR's are no longer blocked. 

I do have a questions about your proposed changes; What benefit would tox add to running the tests in CI? In CI it's already an isolated environment for instance.

**Review by mariobuikhuizen — APPROVED** [review]

**Comment by mariobuikhuizen** [comment]
> As for tox, it may not make that much of a difference in CI but it makes it easier for people running the tests before committing. It also acts as a bit of a Makefile for common commands. I can add it to a separate commit maybe if it's something you're interested in.

We want to keep the tooling the same across our projects for now and use our available resources to enable the users of our libraries as much as possible. Considering changing these tools is not a priority for us right now.

**Comment by vthemelis** [comment]
Fair, let's merge this for now then.

**Comment by mariobuikhuizen** [comment]
Thanks!

**Comment by maartenbreddels** [comment]
Also thanks from me @vthemelis for starting this!

File: setup.py [hunk 1]
```diff
@@ -17,8 +17,7 @@
 from setuptools import setup, find_packages
 
 packages = find_packages()
-module = packages[0]
-# print('module = {}'.format(module))
+module = "ipyaggrid"
```
Comment (maartenbreddels) [inline setup.py:20]: One last comment about this. I think it's find Mario merge it with this, but sometimes small (unrelated) changes can be food for discussion and drag on PR for longer than needed. Especially an important PR like this we want to get in fast. Try to limit your PR's to a single relevant change, thanks 🙏

**Pull request merged**